csatar-theme/pages/tag.htm

```diff
@@ -7,8 +7,16 @@
 paramCode = "code"
 forceSecure = 0
 requirePassword = 0
+
+[basicForm]
+formId = 1
+recordKeyParam = "id"
+createRecordKeyword = "create"
+recordActionParam = "action"
+actionUpdateKeyword = "update"
+actionDeleteKeyword = "delete"
 ==
-<div class="container">
+<div class="container" style="padding-top: 60px;">
```

Comment:
Nem arról volt szó, hogy a stílust külön tesszük be?
Ha ez a menü alá való beszúszást hivatott globálisan megjavítani, akkor nagyon nem jó helyen van.